home-server-stub/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This module will be generated for us. I don't think we need a module for it but we need a module that will generate it.